helm/cas-airflow/values-test.yaml

```diff
@@ -4,6 +4,7 @@
 uid: 1002490001
 gid: 1002490001
 
-# Git sync
-dags:
-  uid: 1002490001
+namespaces:
+  airflow: "0fad32-test"
+  ggircs: "9212c9-test"
+  ciip: "09269b-test"
```

Comment:
```suggestion
```

Those should be retrieved from secrets

Reply:
This is hard to do since we don't have access to the secrets when the helm objects are generated.
`scheduler-roleBinding.yaml` and `worker-roleBinding.yaml` iterate over these namespaces to create rolebindings in predefined namespaces...unless we put these in env variables instead? Or make a container create these?

Reply:
an env variable sounds like a better solution, with an array of namespace prefixes maybe (and the suffix is the same as the release's suffix). 
When shipit will deploy this chart, the env variables can be loaded from a secret

Reply:
I think what @dleard and I envisioned was that shipit would call helm with `--set namespaces.airflow=<<value-from-secret>>`, this way we have an easy way of injecting these when developing.
I suggest for this to just rename `values-dev.yaml` into `values-dev-example.yaml`, etc, to avoid confusion?

Reply:
@pbastia Looks like those can be removed now (in the other values files as well)